Dockerfile

```diff
@@ -0,0 +1,19 @@
+from ruby:2.4.2-jessie
```

Comment:
Does `ruby:2.4.2-alpine3.6` also work?

Reply:
I'm not sure, see my comment about gstreamer.